content/en/notebooks/_index.md

```diff
@@ -128,61 +104,85 @@ To view a different time frame, select an option in the global time picker, or s
 
 ### Template variables
 
-Notebooks support template variables. Dynamically scope visualizations by adding and selecting template variable values. For more information, see [Template Variables][11].
+Notebooks support template variables. Dynamically scope visualizations by adding and selecting template variable values. For more information, see [Template Variables][4].
 
-## Share notebooks
+### Cell configuration
 
-Use the **Share** menu in the upper right of a notebook to see sharing options. Notebooks can be exported to PDF, Markdown, or any document editor.
+To add cells, use the **+** button that appears to the left of the cell. Use the action tray that appears above the cell on hover to share, clone, or delete cells. Graph cells can be exported to a dashboard, or downloaded as a PNG or a CSV of graph data. Changes made in edit mode are saved automatically when you click **Done**. To discard your changes, click **Cancel** instead of **Done**.
 
-{{< img src="notebooks/notebook-export.png" alt="Notebook sharing menu" style="width:50%;">}}
+#### Edit options
+Click on **More options** in the inline editor of a widget to edit widget options. Add details like event overlays, markers, and y-axis controls. 
 
-To copy a notebook into a document editor, click **Copy formatted contents**. Paste into a document editor like Google Docs or Microsoft Word to see notebook contents, including graphs, with original formatting (example below).
+#### Layout options
 
-{{< img src="notebooks/export-to-gdocs.jpeg" alt="Example exported Notebook in Google Docs" style="width:80%;">}}
+In a Notebook cell, click **Edit** to view the cell configuration in edit mode. You can also see the available layout options which vary based on the cell content type but include the following:
 
-### Import or export notebook JSON
+* **Graph size**: Choose between `XS`, `S`, `M` (default), `L`, and `XL`.
+* **Graph legend**: Uncheck the box to hide the legend. Legends are automatically disabled for `XS` and `S` graphs.
+* **Grouping**: Display one graph per tag value to see small multiples of your visualization.
 
-Use **Export Notebook JSON** to download a JSON file containing the definition of your notebook. **Import Notebook JSON** overwrites all content on the notebook with the content of the uploaded JSON.
+{{< img src="notebooks/edit_cell_action_menu.png" alt="layout options for graph size, legend, and grouping configuration" style="width:100%;">}}
+
+**Note**: Changing any of these settings only affects the targeted cell.
+
+#### Types of content
+Notebooks support visualizations and text cells. Text cells are formatted with [Markdown][5], which enables the use of headings, subheadings, links, images, lists, and code blocks.
+
+Graphs in notebooks support all Datadog data sources: metrics, log events, Indexed Spans, live processes, network traffic, RUM events, profiling metrics, and security signals.
```

Comment:
Not sure if this list is meant to be exhaustive, but it might be worthwhile to mention regular events

Reply:
I don't know if this is available, I couldn't see it on my Sandbox account, but I'll look into it. Thanks!